tests/ng-quill.spec.js

```diff
@@ -78,6 +78,7 @@ describe('ng-quill', function () {
       expect(ctrl.module).toBeUndefined()
       expect(ctrl.readOnly).toBeUndefined()
       expect(ctrl.formats).toBeUndefined()
+      expect(ctrl.bounds).toBeUndefined()
```

Comment:
Might it be possible to add an additional test, where you set the bounds through the input?